RSDK-875 migrate rdk proto methods (#1)
viamrobotics/viam-typescript-sdk: README.md

# Viam TypeScript SDK

## (In)stability Notice

> **Warning**

> > This is a beta release of the Viam TypeScript SDK. Stability is not guaranteed. Breaking changes are likely to occur, and occur often.

Reviewer: let's not say beta here at first. this is definitely alpha at this point

Author: ```suggestion
> This is an alpha release of the Viam TypeScript SDK. Stability is not guaranteed. Breaking changes are likely to occur, and occur often.
```
